chalice/deploy/deployer.py

```diff
@@ -280,7 +308,48 @@ def deploy(self, config, existing_resources, stage_name):
                 config, function_name, stage_name)
             deployed_values['api_handler_name'] = function_name
         deployed_values['api_handler_arn'] = function_arn
-        return deployed_values
+
+    def _deploy_auth_handlers(self, config, existing_resources, stage_name,
```

Comment:
Capturing offline:  I'll use either the deployed.json or query lambda based on the expected tags to delete things at the end of the deploy.